documentation/docs/02-template-syntax/05-styles-and-classes.md

```diff
@@ -66,6 +66,36 @@ The `-global-` part will be removed when compiled, and the keyframe will then be
 </style>
 ```
 
+## :global
+
+To apply all styles after a certain point to a selector globally, use the `:global` modifier.
+
+```svelte
+<style>
+	:global {
+		div {
+			/* this will apply to every <div> in your application */
+			margin: 0;
+		}
+		p {
+			/* this will apply to every <p> in your application */
+			color: blue;
+		}
+	}
+
+	div :global strong {
+		/* this will apply to all <strong> elements, in any
+			component, that are inside <div> elements belonging
+			to this component */
+		color: goldenrod;
+	}
```

Comment:
Blocks are still the preferred way to author local-global styles, I think the docs should reflect that. Otherwise it gives the impression that you'd use `:global {...}` for truly global styles, but `.self :global .other` for everything else

```suggestion
	div :global {
		/* this will apply to all <strong> elements, in any
			component, that are inside <div> elements belonging
			to this component */
		strong {...}
	}
```